geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisList.java

```diff
@@ -71,8 +71,7 @@ public RedisList() {
   public RedisList(RedisList redisList) {
     setExpirationTimestampNoDelta(redisList.getExpirationTimestamp());
     setVersion(redisList.getVersion());
```

Comment:
Not really within the scope of this PR, but these two methods are overridable in `AbstractRedisData`, so they shouldn't really be called in the constructor (see https://stackoverflow.com/a/3404369). A simple solution is to make these methods `final` in `AbstractRedisData` so that they're not overridable, which seems reasonable given they're simple setters for private fields of that class.

Reply:
Good call - I've made that change.